tests/test_base32.py

```diff
@@ -17,105 +17,116 @@
 from base32_lib import base32
 
 
-class TestEncode(object):
-    """Encode tests."""
```

Comment:
Should Test classes be avoided altogether or was it for a specific reason here?

Reply:
It's mainly for consistency throughout all modules - normally, test classes have not been used. It's not that they can't be used if there's a good reason, however, in most cases, the pytest fixtures remove the need to have them.